lib/Components/Divider.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class DoubleDivider extends StatelessWidget{
+
+  @override 
+  Widget build(BuildContext context){
+    final instance = ScreenUtil.getInstance();
+    return Row(
+      children: <Widget>[
+        Expanded(
+            child: Divider(
+              color: Colors.grey.shade100,
+            ),
+            
+        ),
+        Padding(
+                    padding: EdgeInsets.fromLTRB(0,
+                  instance.setHeight(20), instance.setWidth(10), instance.setHeight(20)),
+                  ),
+        Text("OR",
```

Comment:
I'd wrap this text in a padding with the padding being 10px to the left and 10px to the right

Reply:
I would recommend also making the OR lowercase like the design doc